src/text-tags.js

```diff
@@ -106,7 +106,7 @@ export function TextTags(seoConfig, config, pageType, data, {url}) {
 
   const twitterTags = seoConfig.enableTwitterCards ? {
     'twitter:card': "summary_large_image",
-    'twitter:title': seoData.title,
+    'twitter:title': pageType === 'story-page' ? get(data, ['data', 'story', 'headline'], 'error') : seoData.title,
```

Comment:
Can we add this in `buildTagsFromStory` function and read from there?

Reply:
Generation of twitter tags is better at a single place, there might not be a significant reason to moving the entire logic to `buildTagsFromStory` because twitter title is independent of a story. Overriding during generation makes it much readable.